integration/gateway/Dockerfile.network

```diff
@@ -1,10 +1,11 @@
 # This Dockerfile runs the RouteGuide server from
 # https://grpc.io/docs/tutorials/basic/python.html
 
-FROM python
+FROM python:3.10.5-slim
 RUN pip install grpcio-tools
-RUN git clone -b v1.14.x https://github.com/grpc/grpc
-WORKDIR grpc/examples/python/route_guide
+ADD https://github.com/grpc/grpc/archive/v1.14.x.tar.gz .
```

Comment:
I think it's ok to pin this to a specific version. I wonder if Dependabot will pick this up and upgrade for us?

Reply:
not sure Dependabot handles grpc or regular github release versioning...